From 8de00e11d693540add7b70f48dd6606d25d872cf Mon Sep 17 00:00:00 2001
From: Frank Chen <FrankChen4@quantatw.com>
Date: Tue, 7 Apr 2026 20:29:39 +0800
Subject: [PATCH] configuration: santabarbara: update NIC thresholds

NIC0 may report negative readings when the OS loads the MLX driver.
This causes the lower critical threshold to be incorrectly triggered.
Remove the lower critical threshold for NIC sensors to avoid false
alarms.

Tested-On: santabarbara
```
root@bmc:~# mfg-tool sensor-display 2>/dev/null | table-sensor-display | grep NIC
NIC0_TEMP_C                   	ok         	43        	DegreesC	N/A	N/A  	N/A	N/A	100  	N/A
```

Change-Id: I0004b92f042415e079ca791b9dad8917e1255f4f
Signed-off-by: Frank Chen <FrankChen4@quantatw.com>
---
 configurations/broadcomm/400g_1p_ocp3.json | 9 +--------
 configurations/broadcomm/800g_1p_ocp.json  | 9 +--------
 configurations/ocp/cx7_ocp.json            | 9 +--------
 configurations/ocp/cx8_ocp.json            | 9 +--------
 4 files changed, 4 insertions(+), 32 deletions(-)

diff --git a/configurations/broadcomm/400g_1p_ocp3.json b/configurations/broadcomm/400g_1p_ocp3.json
index 53074c6..8569e56 100644
--- a/configurations/broadcomm/400g_1p_ocp3.json
+++ b/configurations/broadcomm/400g_1p_ocp3.json
@@ -15,19 +15,12 @@
             "Name": "NIC$bus % 109 % 11 TEMP_C",
             "Name1": "NIC$bus % 109 % 11 TEMP_C",
             "Thresholds": [
-                {
-                    "Direction": "less than",
-                    "Index": 2,
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 5
-                },
                 {
                     "Direction": "greater than",
                     "Index": 2,
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 105
+                    "Value": 100
                 }
             ],
             "Type": "TMP421"
diff --git a/configurations/broadcomm/800g_1p_ocp.json b/configurations/broadcomm/800g_1p_ocp.json
index 767a2a5..3345412 100644
--- a/configurations/broadcomm/800g_1p_ocp.json
+++ b/configurations/broadcomm/800g_1p_ocp.json
@@ -15,19 +15,12 @@
             "Name": "NIC$bus % 109 % 11 TEMP_C",
             "Name1": "NIC$bus % 109 % 11 TEMP_C",
             "Thresholds": [
-                {
-                    "Direction": "less than",
-                    "Index": 2,
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 5
-                },
                 {
                     "Direction": "greater than",
                     "Index": 2,
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 105
+                    "Value": 100
                 }
             ],
             "Type": "TMP421"
diff --git a/configurations/ocp/cx7_ocp.json b/configurations/ocp/cx7_ocp.json
index f9c1965..a083676 100644
--- a/configurations/ocp/cx7_ocp.json
+++ b/configurations/ocp/cx7_ocp.json
@@ -15,19 +15,12 @@
             "Name": "NIC$bus % 109 % 11 TEMP_C",
             "Name1": "NIC$bus % 109 % 11 TEMP_C",
             "Thresholds": [
-                {
-                    "Direction": "less than",
-                    "Index": 2,
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 5
-                },
                 {
                     "Direction": "greater than",
                     "Index": 2,
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 105
+                    "Value": 100
                 }
             ],
             "Type": "TMP421"
diff --git a/configurations/ocp/cx8_ocp.json b/configurations/ocp/cx8_ocp.json
index f858767..45b4870 100644
--- a/configurations/ocp/cx8_ocp.json
+++ b/configurations/ocp/cx8_ocp.json
@@ -15,19 +15,12 @@
             "Name": "NIC$bus % 109 % 11 TEMP_C",
             "Name1": "NIC$bus % 109 % 11 TEMP_C",
             "Thresholds": [
-                {
-                    "Direction": "less than",
-                    "Index": 2,
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 5
-                },
                 {
                     "Direction": "greater than",
                     "Index": 2,
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 105
+                    "Value": 100
                 }
             ],
             "Type": "TMP421"
-- 
2.52.0

